arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Enviroment.kt

```diff
@@ -98,3 +100,15 @@ internal class DefaultEnvironment(override val ctx: CoroutineContext) : Environm
       res.fold(a, e) // Return error to caller
     })
 }
+
+private fun CoroutineContext.asDefaultCtx(): CoroutineContext = plus(DefaultContext(this))
+
+internal class DefaultContext(val ctx: CoroutineContext) : AbstractCoroutineContextElement(DefaultContext) {
+  companion object Key : CoroutineContext.Key<DefaultContext>
+}
+
+internal inline fun CoroutineContext.defaultContext(ctx: CoroutineContext): CoroutineContext =
+  this[DefaultContext] ?: ctx
+
+internal suspend inline fun getDefaultContext(ctx: CoroutineContext = ComputationPool): CoroutineContext =
```

Comment:
Currently, this param is always passed and it's always `ComputationPool`. Should we hardcode it and remove the param?

Reply:
Yeah that could clean this up a bit^^